iceoryx_hoofs/source/posix_wrapper/shared_memory_object.cpp

```diff
@@ -115,7 +115,6 @@ SharedMemoryObject::SharedMemoryObject(const SharedMemory::Name_t& name,
 
     if (m_isInitialized && m_sharedMemory->hasOwnership())
     {
-        std::clog << "Reserving " << m_memorySizeInBytes << " bytes in the shared memory [" << name << "]" << std::endl;
```

Comment:
Can we have this at least as `LogDebug`?

Reply:
Of course, just for you :D